config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Kaminari.configure do |config|
+    config.default_per_page = 5
```

Comment:
Do not add default per page like that.
This should be model specific. So we can have it the flexibility to change this for each. model/table listing.